libs/execution/src/lib/types/valuetypes/standard-representation.ts

```diff
@@ -0,0 +1,103 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import {
+  AtomicValuetype,
+  BooleanValuetype,
+  DecimalValuetype,
+  IntegerValuetype,
+  PrimitiveType,
+  TextValuetype,
+  Valuetype,
+} from '@jvalue/jayvee-language-server';
+
+export class StandardRepresentationResolver {
+  private readonly DECIMAL_COMMA_SEPARATOR_REGEX = /^[+-]?([0-9]*[,])?[0-9]+$/;
```

Comment:
The same regex is also used in `IsValidVisitor`, so consider extracting it as a global constant.

Reply:
Addressed in [c6a0803](https://github.com/jvalue/jayvee/pull/312/commits/c6a08038f19426453a5ad90e66229b861ac2095c)